packages/web-components/src/components/masthead/top-nav-menu.ts

```diff
@@ -36,7 +31,7 @@ class DDSTopNavMenu extends BXHeaderMenu {
   updated(changedProperties) {
     super.updated(changedProperties);
     if (changedProperties.has('active')) {
-      const { active, _triggerNode: triggerNode } = this;
+      const { active, _topMenuItem: triggerNode } = this;
```

Comment:
should this be `_triggerNode` ?

Reply:
yeah it's coming from some changes that the AEM team made 4 weeks ago, so we should keep `_topMenuItem`